object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -91,6 +132,7 @@
    */
   public ObjectConstructionAnnotatedTypeFactory(final BaseTypeChecker checker) {
     super(checker);
+    stubTypes = new StubTypes(this);
```

Comment:
@Nargeshdb you are not actually using this, right?  So we can delete?

Reply:
Yes, I am not using this! It's deleted!